dataset_generation/hi_wikisplit/test_Scraper.py

```diff
@@ -0,0 +1,62 @@
+
+import unittest
+import Scraper as sc
+import os
+import shutil
+
+class TestScraper(unittest.TestCase):
+
+    def test_get_query_params(self):
+        f = open("dummy_file.txt", 'w')
+        test_res = sc.Scraper("dummy_file.txt").get_query_params(['1', '2', '3'])['titles']
+        f.close()
+
+        os.remove("dummy_file.txt")        
+        
+        expected_res = "1|2|3"
+        self.assertEqual(expected_res, test_res)
+
+
+    def test_get_revision_ids(self):
+        f = open("dummy_file.txt", 'w')
+        test_res = sc.Scraper("dummy_file.txt").get_revision_ids('कपूर')
```

Comment:
please rename this file as per its parent file i.e. test_scrape_wiki.py

If the tests are specific for hindi, please rename the file to test_scrape_wiki_hi.py otherwise we can check if the unit tests would work irrespective of language being hindi!

Reply:
yeah, will do that.